src/main/resources/elastic-rest.yml

```
@@ -73,7 +73,7 @@ cat_nodes:
   subdir: "cat"
   versions:
     ">= 0.9.0 < 6.0.0": "/_cat/nodes?v&h=n,nodeId,m,i,r,d,hp,rp,cpu,load_1m,load_5m,load_15m,"
-    ">= 6.0.0": "/_cat/nodes?v&h=n,nodeId,m,i,r,d,hp,rp,cpu,load_1m,load_5m,load_15m&s=n"
+    ">= 6.0.0": "/_cat/nodes?v&h=n,nodeId,i,v,r,m,dup,hp,cpu,load_1m,load_5m,load_15m,iic,sfc,sqc,scc&s=n"
```

I would definitely keep `d` for disk available. `90%` used with 10 GB left is a lot different than `90%` used with 100 GB left.

```suggestion
    ">= 6.0.0": "/_cat/nodes?v&h=n,nodeId,i,v,r,m,d,dup,hp,cpu,load_1m,load_5m,load_15m,iic,sfc,sqc,scc&s=n"
```